src/main/java/dev/naman/userservice/controller/RegistrationController.java

```diff
@@ -35,4 +36,23 @@
         );
     }
 
+    @PostMapping("/user/resetPassword")
+    public ResponseDto<UserResponseDto> resetPassword(@RequestBody UserDto userDto) {
+        User user = userService.resetPassword(userDto);
+        return new ResponseDto<>(
+                HttpStatus.OK,
+                new UserResponseDto(user.getId(), user.getFullName(), user.getEmail(), user.isActive())
```

Comment:
Assume while logging in, person clicks on reset password. Then a new page opens that asks your email. So the only thing that will be sent in request will be the email ID of the person.

Reply:
sure. I'll correct that.